service/controller/key/amis.go

```diff
@@ -1,249 +1,147 @@
+// NOTE: This file is generated. Do not edit.
```

Comment:
This seems to still be missing `china` images. Can this easily be extended to include them?

Reply:
We generate as an artifact of the build to China a JSON file containing the `current` (latest) AMIs ID for China. It is public available

Reply:
Yep! I'll add it in a separate PR once I have the China JSON URL.